loggins.sh

```diff
@@ -5,31 +5,40 @@
 autoload -U add-zsh-hook
 
 # Start derby
+source load.zsh &> /dev/null
```

Comment:
I don't think loggins should be something manually started. The logging feature should be enabled by virtue of the user sourcing `logging/load.zsh`.